CourseController/src/views/Admin.vue

```diff
@@ -27,21 +26,8 @@ const toggleAdmin = () => {
       </Teleport>
     </template>
     <template #main>
-      <div class="flex flex-col gap-[20dvh]">
-        <AdminNavbar v-model:isAdmin="isAdmin" />
-        <div class="flex flex-col items-center justify-center backdrop-blur-md shadow-figmaxl py-[3dvh] rounded-3xl">
-          <div class="flex flex-col gap-[5dvh] mb-[5dvh]">
-            <p class="place-self-center">Modulok</p>
-            <div class="grid grid-cols-2 gap-[3dvh_5dvw]">
-              <AdminModule content="Szerkesztes" />
-              <AdminModule content="Bejegyzes" />
-              <AdminModule content="Feltoltes" />
-              <AdminModule content="Ora" />
-              <Button label="Switch admin" icon="pi pi-cog" @click="toggleAdmin"  />
-            </div>
-          </div>
-        </div>
-      </div>
+      <router-view/>
```

Comment:
Important, `<router-view>` and `<RouterView>` is not the same, there are some interoperability issues, or smth ask @AFekexd

Reply:
donre